src/tools/execlog/src/main/java/com/google/devtools/build/execlog/ExecLogParser.java

```diff
@@ -130,9 +132,22 @@ public Element(int position, SpawnExec element) {
     // actions in input that are not in the golden, in order received.
     Queue<SpawnExec> uniqueActions;
 
+    // use regex expression to find shard value in file
+    private int getShard(SpawnExec ex) {
+      Pattern pattern = Pattern.compile("shard_[1-9][0-9]?");
```

Comment:
Not sure if the shard number can go over 100, but can we make this pattern more general and robust in case it does?

Reply:
Made it into 1 - 1000.